### DISCUSSION TITLE
How to get latent representations of all training samples during pretraining

### DISCUSSION THREAD

**Discussion by zhaozh10** [open-mmlab/mmselfsup#642] [How-to]
I hope to observe the change in each sample's latent representation every epoch, and I hope to get the representations obeying the order of the dataset. How can I implement it in mmselfsup?

**Comment by YuanLiuuuuuu** [comment]
In ``loss`` func (which is defined in the algorithm module in MMSelfSup ``1.x`` branch), after you obtain these features, you can directly save them to your disk for further analysis.
